src/uct/api/tl.h

```diff
@@ -347,6 +350,7 @@ typedef struct uct_iface_ops {
     uct_ep_destroy_func_t               ep_destroy;
     uct_ep_get_address_func_t           ep_get_address;
     uct_ep_connect_to_ep_func_t         ep_connect_to_ep;
+    uct_ep_connect_to_ep_v2_func_t      ep_connect_to_ep_v2;
```

Comment:
it should be in uct_iface_internal_ops_t to not break ABI

Reply:
it's because this struct is H file, and used from inline functions, adding new fields moves existing fields
anyway, in v2 slowpath functions are internal struct